raster/raster-layer-geopackage/README.md

```diff
@@ -1,23 +1,34 @@
-# Raster Layer GeoPackage
+# Raster layer (GeoPackage)
 
-Display raster data from a geopackage.
+Display a raster contained in a GeoPackage.
 
-![](RasterLayerGeopackage.png)
+![Image of raster layer GeoPackage](RasterLayerGeopackage.png)
 
-## How it works
+## Use case
+
+GeoPackages are designed to simplify file management and transfer. They can store raster files (as well as other types of data). An end-user wishing to transfer rasters from ArcGIS Pro or between runtime apps might need to import raster files from GeoPackages into their map to view and analyze the data.
+
+## How to use the sample
 
-To add a`RasterLayer` as an operational layer from a `GeoPackage`:
+When the sample starts, a raster will be loaded from a GeoPackage and displayed in the map view.
+
+## How it works
 
 1. Create and load a `GeoPackage`, specifying the path to the local .gpkg file.
-2. When it is done loading, get the `GeoPackageRaster`s inside with `geoPackage.getGeoPackageRasters()`.
-3. Construct a `RasterLayer` with the `GeoPackageRaster` in the list you want to use.
-4. Add the raster layer to the map as an operational layer `map.getOperationalLayers().add(rasterLayer)`.
+2. When it is done loading, get the `GeoPackageRaster`s from the geopackage.
+3. Construct a `RasterLayer` with the geopackage raster you want to use.
```

Comment:
Shuld these not say 'GeoPackage' since it's a named term? We're also writing 'GeoPackage' in all the sections above

```suggestion
2. When it is done loading, get the `GeoPackageRaster`s from the GeoPackage.
3. Construct a `RasterLayer` with the GeoPackage raster you want to use.
```